config.json

```diff
@@ -180,6 +180,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "StrictPrerequisiteVersionMatchAdditionalArg": {
```

Comment:
Similar to my other comments is I think I would replace "Prerequisite" with "tools" in the various places. Maybe it is just me but I think tools is much easier to read and type.